python/setup.py

```diff
@@ -0,0 +1,6 @@
+from setuptools import setup, find_packages
+
+setup(name='festival_testing', 
```

Comment:
Use a different name. Instead of `festival`, choose one of the following:  

- `tts`
- `techmo_tts`
- `tribune`
- `tribune_tts`
- `flavia`
- `flavia_tts`

Maybe ask @tomjad which one is preferred.